docs/admin/20-getting-started/20-docker-compose.md

```diff
@@ -5,84 +5,185 @@ title: Docker Compose
 description: "🌟 Full-blown featureset including web office and full-text search."
 ---
 
-# Docker Compose
+# OpenCloud with Docker Compose
 
+Install a internet facing OpenCloud with SSL certification with Docker Compose.
 
-Spin up a temporary local instance of OpenCloud using **Docker Compose**.
+## **Prerequisites**
 
+- **Four domains** pointing to your server:
+  - `cloud.YOUR.DOMAIN` → OpenCloud frontend
+  - `collabora.YOUR.DOMAIN` → Collabora Online Server
+  - `wopiserver.YOUR.DOMAIN` → WOPI server for document editing
+  - `traefik.YOUR.DOMAIN` → Traefik dashboard
 
-## **Prerequisites:**
-- **Linux**, **Mac** or **Windows** Subsystem for Linux [(WSL)](https://learn.microsoft.com/en-us/windows/wsl/install)
-- [**Git**](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [**Docker Compose**](https://docs.docker.com/compose/install/)
+  Alternatively, you can use a wildcard domain (`*.YOUR.DOMAIN`)
+- A **hosted server** (e.g., Hetzner, AWS, or your own VPS) with Linux and SSH access
 
 ---
 
-##  1. Download
+## 1. Connect to Your Server
+Log into your server via SSH:
 
-Download the `opencloud_full` folder (this folder contains a multi-file Docker Compose configuration):
+```bash
+ssh root@YOUR.SERVER.IP
+```
 
-```Shell
-git clone https://github.com/opencloud-eu/opencloud.git
+## 2. Install Docker
+Update your system and install Docker.
+
+
+First, update your package list:
+
+```bash
+apt update && apt upgrade -y
 ```
+Install Docker following the [official Docker guide](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
```

Comment:
Again a reference to Ubuntu. Either it is targeted at Ubuntu, than we should change the Prerequisite above saying that, or we keep it distro agnostic.